EventFilter/HcalRawToDigi/python/HcalDigiToRawuHTR_cfi.py

```diff
@@ -0,0 +1,11 @@
+import FWCore.ParameterSet.Config as cms
+
+hcalRawDatauHTR = cms.EDProducer("HcalDigiToRawuHTR",
```

Comment:
there are two issues with this file now (sorry for not noticing sooner)
-  the cfi file name should better match the instance of the producer defined in it (hcalRawDatauHTR_cfi.py instead of HcalDigiToRawuHTR_cfi.py)
- given that the fillDescriptions are now provided, the file is now superfluous. The framework  auto-generates hcalDigiToRawuHTR_cfi.py (available as EventFilter.HcalRawToDigi.hcalDigiToRawuHTR_cfi)

So, I suggest to remove the file.